flask_jwt_extended/tokens.py

```diff
@@ -127,12 +126,15 @@ def decode_jwt(encoded_token, secret, algorithm, identity_claim_key,
     :param audience: expected audience in the JWT
     :param leeway: optional leeway to add some margin around expiration times
     :param allow_expired: Options to ignore exp claim validation in token
+    :param verify_audience: Options to ignore aud claim validation in token
     :return: Dictionary containing contents of the JWT
     """
     options = {}
     if allow_expired:
         options['verify_exp'] = False
 
+    options['verify_aud'] = verify_audience
```

Comment:
This could be moved into the `options` constructor, as it will always be an argument that is passed in.